src/app/key-concepts/natural-capital-and-biodiversity/page.tsx

```diff
@@ -0,0 +1,214 @@
+                  className="h-[224px] w-[227px] xl:h-[357px] xl:min-h-[357px] xl:w-[347px] xl:min-w-[347px]"
+                />
+                <div className="absolute inset-0 flex h-full w-full items-center justify-center">
+                  <figcaption className="h-14 rounded-[40px] bg-grey-800/80 px-6 py-4 text-center text-white">
+                    Water cycling
+                  </figcaption>
+                </div>
+              </figure>
+              <figure className="relative">
+                <Image
+                  alt="Soil formation"
+                  src="/assets/home-pebble-1.png"
+                  width={347}
+                  height={357}
+                  className="h-[224px] w-[227px] xl:h-[357px] xl:min-h-[357px] xl:w-[347px] xl:min-w-[347px]"
+                />
+                <div className="absolute inset-0 flex h-full w-full items-center justify-center">
+                  <figcaption className="h-14 rounded-[40px] bg-grey-800/80 px-6 py-4 text-center text-white">
+                    Soil formation
+                  </figcaption>
+                </div>
+              </figure>
+            </div>
+          </div>
+          <div className="flex w-full items-center justify-center">
+            <div className="flex flex-col items-center gap-6 lg:w-[559px] lg:gap-20"></div>
+          </div>
+          <div className="w-full bg-pink bg-cover p-6 py-10 lg:bg-[url(/assets/climate-change-background-imperative.png)] lg:p-10 lg:bg-blend-luminosity">
+            <div className="flex flex-col gap-[25px]">
+              <div className="max-w-[790px] text-2xl lg:text-4xl">
+                However, natural capital is more than just biodiversity.
+              </div>
+              <div className="flex max-w-[790px] flex-col gap-2">
+                <p>
+                  It includes the emergent properties that arise from the interaction of species and
+                  the combination of ecosystem and abiotic resources and also the structures,
+                  functions, and processes of ecosystems that support human well-being and economic
+                  activities.
+                </p>
+                <p className="font-bold">Example of natural capital, biodiversity and beyond</p>
+                <p>
+                  Ecosystem services such as pollination, pest control, and the production of food
+                  and fiber depend directly on biodiversity.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div className="flex flex-col gap-6 self-start text-grey-800 max-lg:pb-5 lg:max-w-[799px]">
+            <div className="text-2xl tracking-tight lg:max-w-[737px] lg:text-4xl">
+              We need to protect both for sustainability for two reasons:
+            </div>
+          </div>
+          <div className="flex w-full items-center justify-center">
+            <div className="flex flex-col items-center gap-10 lg:w-[559px] lg:gap-20">
```

Comment:
Use a similar layout as the “Natural Capital and Biodiversity” section below and the one from the homepage: https://github.com/Vizzuality/natural-capital-primer/blob/b91e0428a4365fa75e42b0f7264c41ff3f40cf56/src/app/page.tsx#L178.

That way we're consistent on this page and consistent with the other pages.